src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexer.cs

```diff
@@ -600,6 +605,12 @@ public AddressBalancesResult GetAddressBalances(string[] addresses, int minConfi
         /// <inheritdoc />
         public VerboseAddressBalancesResult GetAddressIndexerState(string[] addresses)
         {
+            // If the containing feature is not initialized then wait a bit.
+            if (!this.storeSettings.AddressIndex)
+                throw new NotSupportedException("Address indexing is not supported.");
```

Comment:
Maybe 'not enabled' would be more informative than not supported

Reply:
Ditto what @zeptin  said 👍